client/extensions/woocommerce/state/ui/payments/methods/selectors.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { get, filter, find, findIndex, remove } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getApiPaymentMethods, arePaymentMethodsLoaded } from 'woocommerce/state/sites/payment-methods/selectors';
+
+const getPaymentMethodsEdits = ( state, siteId ) => {
+	return get( state, [ 'extensions', 'woocommerce', 'ui', 'payments', siteId, 'methods' ] );
+};
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {Array} All changes made to method
+ */
+export const getPaymentMethodEdits = ( state, siteId ) => {
+	return get( state, [ 'extensions', 'woocommerce', 'ui', 'payments', siteId, 'methods', 'currentlyEditingChanges' ] );
+};
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {Array} The list of payment methods that the UI should show. That will be the list of methods returned by
+ * the wc-api with the edits "overlayed" on top of them.
+ */
+export const getPaymentMethods = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
ah, I suppose this is why you had `Api` in the sites selectors. What about `getPaymentMethodsWithEdits`?